lib/common/utils.c

```diff
@@ -343,11 +343,19 @@ crm_generate_uuid(void)
     return buffer;
 }
 
+static void
+_gnutls_log_func(int level, const char *msg)
+{
+    crm_trace("%s", msg);
+}
+
 void
 crm_gnutls_global_init(void)
 {
     signal(SIGPIPE, SIG_IGN);
     gnutls_global_init();
+    gnutls_global_set_log_level(9);
```

Comment:
9 still shows private keys, 8 would be fine (though it looks like they don't actually use 7 or 8 currently)